doc/cupertino-getting-started.md

```diff
@@ -123,36 +146,35 @@ If your application is based on the older solution template that includes a shar
     </ResourceDictionary>
     ```
 
-4. In `AppResources.xaml`, update `<CupertinoColors />` with the override from the previous steps:
+3. In `App.xaml`, update `<CupertinoColors />` with the override from the previous steps:
 
     ```xml
     <CupertinoColors xmlns="using:Uno.Cupertino"
-                     OverrideSource="ms-appx:///PROJECT_NAME/Style/Application/CupertinoColorsOverride.xaml" />
+                     OverrideSource="ms-appx:///Styles/Application/CupertinoColorsOverride.xaml" />
```

Comment:
@kazo0 depending on the type of project template that the Uno Platform application was created with, this could be different right? (same question for the other similar changes below where `PROJECT_NAME` is removed

Reply:
yeah this part im just assuming we are documenting for the new template method, do we want to have another tabbed section here too?

Reply:
good question for @jeromelaban :)

Reply:
Fyi that'd mean covering each template in both material and Cupertino for installation, font overrides, and color overrides 

Which I feel a bit overkill

Reply:
Yeah you are maybe right...

Reply:
We are documenting for the new template. You can point to resources management here, though: https://platform.uno/docs/articles/guides/how-to-create-control-libraries.html#moving-the-control-style-in-a-separate-resource-dictionary